test/integration/targets/azure_rm_virtualmachine/tasks/virtualmachine.yml

```diff
@@ -81,7 +89,9 @@
       admin_password: Password123!
       short_hostname: testvm
       os_type: Linux
-      network_interfaces: testvm001
+      network_interfaces:
```

Comment:
@yuwzho Am I inserting a list of nics correctly here? Usually do this via a variable containing a list...